chipsec/cfg/8086/pch_6xxP.xml

```diff
@@ -32,6 +32,8 @@ XML configuration file for 6XX series pch
   <info>
     <sku did="0x5182" name="6xxP" code="PCH_6XXP" longname="Intel ADL-P (600) PCH" />

```

Comment:
Isn't this one included in the first range?

Reply:
I just removed both of them as they're a part of the range.